File: examples/js/post__predict_anomalies.js
// 示例：POST /predict/anomalies
fetch('http://localhost:8080/api/v1/predict/anomalies', {
    method: 'POST'
})
.then(res => res.json())
.then(data => console.log(data))
.catch(err => console.error('Request error:', err));

File: examples/python/post__predict_anomalies.py
# -*- coding: utf-8 -*-
import requests

# 示例：POST /predict/anomalies
# 使用 POST 方法请求 /predict/anomalies
url = "http://localhost:8000/api/v1/predict/anomalies"
headers = {"Authorization": "Bearer YOUR_TOKEN"}

try:
    response = requests.post(url, headers=headers)
    print("Status:", response.status_code)
    print("Response:", response.json())
except Exception as e:
    print("Request failed:", e)
